packages/core-util/src/sdks/colorPalette.ts

```diff
@@ -0,0 +1,19 @@
+// color palette copied from https://cloudscape.design/foundation/visual-foundation/data-vis-colors/
+export const colorPalette = [
+  '#688ae8',
```

Comment:
is it possible to just take their JS variables?

Reply:
No we can't. I tried to import their variables. But they export variables like 
``` 
export var colorChartsPaletteCategorical1 = "var(--color-charts-palette-categorical-1-bonb16, #688ae8)";
```
which won't work with our charts.  So we have to use hard coded strings.

Reply:
ahhh dang, makes sense!